group.go

```diff
@@ -169,15 +161,11 @@ func (client *Client) UpdateGroup(input GroupUpdateInput) (*Group, error) {
 
 func (client *Client) DeleteGroupWithAlias(alias string) error {
 	var m struct {
-		Payload struct {
-			Id     graphql.ID       `graphql:"deletedId"`
-			Alias  graphql.String   `graphql:"deletedAlias"`
-			Errors []OpsLevelErrors `graphql:"errors"`
-		} `graphql:"groupDelete(input: $input)"`
+		Payload ResourceDeletePayload `graphql:"groupDelete(resource: $input)"`
 	}
 	v := PayloadVariables{
-		"input": ResourceDeletePayload{
-			Alias: alias,
+		"input": IdentifierInput{
```

Comment:
Should NewIdentifier func be used here too?

Reply:
fixed